automation/src/test/java/org/wso2/testgrid/automation/executor/TestNgResultsParserTest.java

```diff
@@ -44,27 +44,35 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.nio.file.StandardCopyOption;
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Optional;
 
 public class TestNgResultsParserTest {
 
     private static final String TESTGRID_HOME = Paths.get("target", "testgrid-home").toString();
     private static final String SUREFIRE_REPORTS_DIR = "surefire-reports";
+    private static final String SUREFIRE_REPORTS_DIR_OUT = "test-outputs/scenarios/SolutionPattern22/surefire-reports";
```

Comment:
`test-outputs/scenarios` can be replaced with `TestGridConstants.TEST_RESULTS_DIR`